lms/static/scripts/frontend_apps/components/dashboard/test/AllCoursesActivity-test.js

```diff
@@ -96,6 +95,24 @@ describe('AllCoursesActivity', () => {
     wrapper.update();
   }
 
+  [
+    { organization: undefined, expectedTitle: 'All courses' },
+    {
+      organization: {
+        name: 'University of Hypothesis',
+      },
+      expectedTitle: 'University of Hypothesis',
+    },
+  ].forEach(({ organization, expectedTitle }) => {
+    it('sets expected section title', () => {
```

Comment:
Should "section title" be "page title" perhaps?

Reply:
Yes, makes sense